### DISCUSSION TITLE
osu crash on amd graphic card when I minimize and reopen the game. It happens almost all time

### DISCUSSION THREAD

**Discussion by Neon095** [ppy/osu#24927] [Q&A]
``### Type

Crash to desktop

### Bug description

When I have lazer open and I change screens for a few secs, when I reopen the game It crashes to desktop and also the graphic card crashes.. It happens a lot.

### Screenshots or videos

![image](https://github.com/ppy/osu/assets/97938542/93b2d86a-7f0c-464d-802e-fbc491246c8e)
[video.zip](https://github.com/ppy/osu/files/12717268/video.zip)

### Version

2023.924.1-lazer

### Logs

[network.log](https://github.com/ppy/osu/files/12717141/network.log)
[performance.log](https://github.com/ppy/osu/files/12717142/performance.log)
[runtime.log](https://github.com/ppy/osu/files/12717143/runtime.log)
[updater.log](https://github.com/ppy/osu/files/12717144/updater.log)
[database.log](https://github.com/ppy/osu/files/12717145/database.log)
[input.log](https://github.com/ppy/osu/files/12717146/input.log)

**Comment by peppy** [comment]
Could you try updating your AMD drivers, and also check event log for driver related crashes?

**Reply by Neon095** [reply]
I have the latest drivers

**Comment by Neon095** [comment]
I have the latest AMD drivers (23.9.2)
[atikmdag_dce - Copy.log](https://github.com/ppy/osu/files/12727692/atikmdag_dce.-.Copy.log)
